source/documentation/services/airflow/index.md

```diff
@@ -174,7 +196,7 @@ dag:
   repository: moj-analytical-services/analytical-platform-airflow-python-example
   tag: 2.0.0
   env_vars:
-    FOO: "bar"
+    x: "1"
```

Comment:
Trying to keep consistent to the multi task below where I have used diff variables (more data engineering/sciency)